lale/lib/sklearn/stacking_classifier.py

```diff
@@ -0,0 +1,260 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pandas as pd
+from sklearn.ensemble import StackingClassifier as SKLModel
+
+import lale.docstrings
+import lale.operators
+
+from .stacking_utils import _concatenate_predictions_pandas
+
+
+class _StackingClassifierImpl(SKLModel):
+    def predict(self, X):
+        return super().predict(X)
+
+    def predict_proba(self, X):
+        return super().predict_proba(X)
+
+    def score(self, X, y, sample_weight=None):
+        return super().score(X, y, sample_weight)
+
+    def decision_function(X):
+        return super().decision_function(X)
+
+    def _concatenate_predictions(self, X, predictions):
+        if not isinstance(X, pd.DataFrame):
+            return super()._concatenate_predictions(X, predictions)
+        return _concatenate_predictions_pandas(self, X, predictions)
+
+
+_hyperparams_schema = {
+    "description": "Stack of estimators with a final classifier.",
+    "allOf": [
+        {
+            "type": "object",
+            "required": [
+                "estimators",
+                "final_estimator",
+                "cv",
+                "stack_method",
+                "n_jobs",
+                "passthrough",
+            ],
+            "relevantToOptimizer": [
+                "estimators",
+                "final_estimator",
+                "cv",
+                "passthrough",
+            ],
+            "additionalProperties": False,
+            "properties": {
+                "estimators": {
+                    "type": "array",
+                    "items": {
+                        "type": "array",
+                        "laleType": "tuple",
+                        "items": [
+                            {"type": "string"},
+                            {"anyOf": [{"laleType": "operator"}, {"enum": [None]}]},
+                        ],
+                    },
+                    "description": "Base estimators which will be stacked together. Each element of the list is defined as a tuple of string (i.e. name) and an estimator instance. An estimator can be set to ‘drop’ using set_params.",
+                },
+                "final_estimator": {
+                    "anyOf": [{"laleType": "operator"}, {"enum": [None]}],
+                    "default": None,
+                    "description": "A classifier which will be used to combine the base estimators. The default classifier is a 'LogisticRegression'",
+                },
+                "cv": {
```

Comment:
To resolve this TODO, I think we can specify the "cv" of StackingClassifier similarly to the "cv" of [Hyperopt](https://github.com/IBM/lale/blob/master/lale/lib/lale/hyperopt.py#L476).